app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -257,12 +257,12 @@ $(document).on("ready", function() {
     var tool = $(this).parents(".tool-body").find(".congress-message-tool-container");
     var emailValues = tool.data("email-values");
 
-    if (tool.data("target-bioguide-id")) {
+    if (tool.data("bioguide-ids").length > 0) {
       emailValues["$MESSAGE"] = form.find(".campaign-message").val();
 
       $("form.congress-message-rep-lookup").html("");
 
-      var bioguide_ids = tool.data("bioguide-id").split(",");
+      var bioguide_ids = tool.data("bioguide-ids").split(",");
```

Comment:
The pattern here needs to include possible spaces around the comma since that's how `CongressMessageHelper` joins the array. Otherwise only the first bioguide_id is located by phantomdc since the others are prefixed by a space. The error passes very silently but you can test by making sure that multiple `fieldset.congressForms-legislator-fields` appear when multiple bioguide ids are given.

Reply:
Thanks for catching this! Looks like it's an issue for actions that are configured to always use specific bioguide IDs as well.

I can't find a ways to pass puffy billy a post body in the stubbed phantomdc, so the integration test won't catch it (it always returns the same response for the form fields lookup endpoint). With caching enabled, puffing billy saves a different response for different post bodies, so we can check for that once #380 is done. I'll make a note in that ticket.